### PR TITLE
Added ability for torchbench to read from a file

### PR DESCRIPTION
**Pull request by Chillee**
`time ./benchmarks/torchbench.py -dcuda --float16 --inductor --training --batch_size_file /scratch/chilli/fresh/torchdynamo/benchmarks/torchbench_models_list.txt`

### CODE REVIEW

File: benchmarks/torchbench.py [hunk 1]
```diff
@@ -272,6 +273,17 @@ def load_model(
         benchmark_cls = getattr(module, "Model", None)
         if not hasattr(benchmark_cls, "name"):
             benchmark_cls.name = model_name
+        batch_size = 1
+        lines = open('/fsx/users/chilli/model_sizes.txt', 'r').readlines()
```
Comment (anijain2305) [inline benchmarks/torchbench.py:277]: Can we put the file in torchdynamo repo itself?

**Comment by ngimel** [comment]
What would be the way to run with default batch sizes?

**Comment by Chillee** [comment]
@ngimel Hmm... no way here. I guess I'll add an additional flag.

**Comment by Chillee** [comment]
Added `--batch_size_file` flag, which allows torchbench to read the batch size from a file.

File: benchmarks/common.py [hunk 2]
```diff
@@ -1449,14 +1452,35 @@ def main(runner, original_dir=None):
     experiment = functools.partial(experiment, args, model_iter_fn)
 
     if args.only:
```
Comment (ngimel) [inline benchmarks/common.py:1454]: how does this work when args.only is false?

Comment (Chillee) [reply benchmarks/common.py:1454]: Well, if you're running with `isolate` (which I think we always should), it just ends up calling `--only`.

Comment (ngimel) [reply benchmarks/common.py:1454]: Can we check then that if `--batch-size-file` is set, either `--isolate` or `--only` is set too?

Comment (Chillee) [reply benchmarks/common.py:1458]: Updated.

**Review by ngimel — APPROVED** [review]

**Pull request merged**